python/cuml/cuml/tests/test_spectral_embedding.py

```diff
@@ -0,0 +1,260 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+from urllib.request import urlretrieve
+
+import cupy as cp
+import h5py
+import numpy as np
+import pytest
+from sklearn.datasets import fetch_openml, make_s_curve
+from sklearn.manifold import SpectralEmbedding as skSpectralEmbedding
+from sklearn.manifold import trustworthiness
+from sklearn.model_selection import train_test_split
+
+from cuml.manifold import SpectralEmbedding
```

Comment:
sklearn exposes a functional `spectral_embedding` api as well. Since you defined that function, would you mind testing it here too (a smoketest is fine if the class test is more sufficient)?